### PR TITLE
remove print statements

### PR DESCRIPTION
**Pull request by couture-ql**
Removes pointless print statements from datatable model.

While we're at it bump the version.

### CODE REVIEW

**Comment by couture-ql** [comment]
Turns out our dependencies for pandas and numpy, which start out very old, require upper bounds at the moment for python 3.5.  The current patch caps those changes to other platforms as well, which isn't very good.

We can introduce branching in setup.cfg in the mean-time. I may explore that.  However, it's not really worth it as 3.5 is  EOL soon and we should drop it ourselves.

File: quandl/model/datatable.py [hunk 1]
```diff
@@ -36,7 +36,6 @@ def download_file(self, file_or_folder_path, **options):
         while not file_is_ready:
             file_is_ready = self._request_file_info(file_or_folder_path, params=options)
             if not file_is_ready:
-                print(Message.LONG_GENERATION_TIME)
```
Comment (jjmar) [inline quandl/model/datatable.py:39]: This message is actually pretty useful. As generating a bulk download file may take a bit of time having this message be printed on repeat will help reassure users that the script is not hanging.

Ideally we replace these with Logger calls instead

Comment (couture-ql) [reply quandl/model/datatable.py:39]: I feel like we should introduce that in another PR, which adequately introduces sane logging.

Comment (couture-ql) [reply quandl/model/datatable.py:39]: Though if we can have a consensus sooner, it could be introduced in this series as a replacement.

Comment (couture-ql) [reply quandl/model/datatable.py:39]: I've changed my mind; I'll introduce it in this series, but it's going to be lame.  I would have preferred we spend the time to introduce log.debug() in a few more places that make sense.  For now, we'll just replace print() with debug()

**Pull request merged**